pkg/virt-handler/selinux/labels.go

```diff
@@ -53,10 +53,11 @@ func (se *SELinuxImpl) IsPresent() (present bool, mode string, err error) {
 	if err != nil {
 		return false, string(out), err
 	}
-	if strings.Contains(string(out), "disabled") {
-		return false, string(out), nil
+	outStr := strings.TrimSpace(string(out))
```

Comment:
What about the conversion to lower case?

`getenforce` returns `Permissive` and I'm not seeing any ToLower(), so I would be surprised if `outStr == "disabled"` works.

Reply:
the code is here: https://github.com/kubevirt/kubevirt/pull/2694/files#diff-7ff8c074056848ec8628c964418672bfR13, the PR is not using the `getenforce` binary anymore.

Reply:
Oh right I missed that, it looks like trimming would also be unnecessary then.

Reply:
Hm, it still writes a "newline" which we need to trim.